### PR TITLE
Add rollback functionality

### PR DESCRIPTION
**Pull request by gargshobhit90**

### CODE REVIEW

File: bin/out [hunk 1]
```diff
@@ -111,15 +118,21 @@ fi
...
-# do things
-case $RESOURCE_TYPE in
-    deployment)
-    deploy "$RESOURCE_NAME" "$CONTAINER_NAME" "$IMG" "$DEPLOYMENT_PATH" "$IMG_PREFIX";;
-    job)
-    start_job "$RESOURCE_PATH" "$IMG" "$(date +%s)";;
-    *)
-    exit 1
-esac
+if [ "$ROLLBACK" == "true" ]; then
```
Comment (dharmesh-periscope) [inline bin/out]: Will it be better if RESOURCE_TYPE is the outer condition?

**Comment by dharmesh-periscope** [comment]
Looks fine for the most part. Certainly needs to be tested out.

File: bin/out (hunk 1 above)
Comment (ilge) [reply bin/out]: +1

File: bin/out [hunk 2]
```diff
@@ -111,15 +118,20 @@ fi
...
-case $RESOURCE_TYPE in
-    deployment)
-    deploy "$RESOURCE_NAME" "$CONTAINER_NAME" "$IMG" "$DEPLOYMENT_PATH" "$IMG_PREFIX";;
-    job)
-    start_job "$RESOURCE_PATH" "$IMG" "$(date +%s)";;
-    *)
-    exit 1
-esac
+if [[ "$RESOURCE_TYPE" = 'deployment' ]]; then
+    if [[ -z "$ROLLBACK" -o "$ROLLBACK" = "false" ]]; then
```
Comment (ilge) [inline bin/out]: json parsed ROLLBACK param will become "null" if it's absent. Therefore we're better off checking for presence to do rollback than absence to not do it.

**Pull request merged**